src/FetchDataFromGitLabUserCountsCommand.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Creacoon\GitLabTile;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Http;
+
+class FetchDataFromGitLabUserCountsCommand extends Command
+{
+    protected $signature = 'dashboard:fetch-data-from-gitlab-api';
+    protected $description = 'Fetch data for GitLab tile';
+
+    protected $specificUsers = [];
+
+    public function handle()
+    {
+        $gitlabConfig = config('dashboard.tiles.gitlab');
+        $specificUsers = $gitlabConfig['specific_users'];
```

Comment:
Please use the configs directly, this helps them to be found, and you can easily click through in the IDE.
So e.g. `config('dashboard.tiles.gitlab.specific_users')`.